src/app/backend/resource/pod/detail.go

```diff
@@ -46,7 +46,7 @@ type PodDetail struct {
 	NodeName                  string                                          `json:"nodeName"`
 	RestartCount              int32                                           `json:"restartCount"`
 	QOSClass                  string                                          `json:"qosClass"`
-	Controller                controller.ResourceOwner                        `json:"controller"`
+	Controller                *controller.ResourceOwner                       `json:"controller,omitempty"`
```

Comment:
I think this is breaking the tests in detail_test.go

See https://travis-ci.org/kubernetes/dashboard/jobs/496515775#L364

Reply:
Sure, didn't check it before. Thanks :)